examples/lua/prometheus_adapter.lua

```diff
@@ -0,0 +1,16 @@
+local http = require("socket.http")
+local ltn12 = require("ltn12")
+
+local respbody = {}
+local field = "rgw_request_content_length " .. tostring(Request.ContentLength) .. "\n"
```

Comment:
you should probably put that inside an "if" statement checking that this is a "put" (or "get"?) request
otherwise, your measurements will be biased due to "delete" (or other operations) that have small content